### PR TITLE
Being honest about Embodied Carbon from Software Development

### PR DESCRIPTION
**Pull request by jhowlett-scottlogic**
Hi Colin, 

this is my second post and I have had the content reviewed by Oliver Cronk 

Please add a direct link to your post here:

https://jhowlett-scottlogic.github.io/blog/2023/09/26/embodied-carbon-from-software-development.html

Have you (please tick each box to show completion):

 - [x] Added your blog post to a single category?
 - [x] Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
 - [x] Checked that the build passes?
 - [x] Checked your spelling (you can use `npm spellcheck` if that's your thing)
 - [x] Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
 - [x] Optimised any images in your post? They should be less than 100KBytes as a general guide.

Posts are approved based on their category. replace `(at)` with `@` to notify them!

 - Tech - Colin Eberhardt - @ColinEberhardt
 - Cloud - Colin Eberhardt - (at)ColinEberhardt
 - Delivery - Matt Phillips - (at)scottlogicmphillips
 - UX - Graham Odds - (at)godds
 - People - Graham Odds - (at)godds
 - Testing - Laurence Pisani - (at)lpisani-SL

### CODE REVIEW

File: _posts/2023-09-26-embodied-carbon-from-software-development.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Sustainability
+summary: Carbon emissions come in all shapes and sizes, in this blog post I talk about the more illusive sources of embodied carbon from software development.
+author: jhowlett
+---
+
+### The software industry has had an easy ride compared with other sectors regarding sustainability, in the coming years that could all be about to change.
+
+I wanted to build upon my previous post [A guide to Software Sustainability terminology](https://blog.scottlogic.com/2023/09/12/sustainability-terminology.html) and expand on a term that I feel does not have the awareness it deserves. If you are new to this topic then I recommend giving it a read first.
+
+The software industry has only had minimal, if any imposing regulations or mandatory audits of sustainability practices. This could be due to the media portraying the digital revolution as part of the green future, software having grown and evolved so fast that regulatory bodies cannot keep up or maybe even just because computers are not directly spewing out plumes of harmful carbon emissions.
```
Comment (kietSL) [inline _posts/2023-09-26-embodied-carbon-from-software-development.md]: comma after "if any"

File: _posts/2023-09-26-embodied-carbon-from-software-development.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Having a greater awareness and software specific definition of embodied carbon allows better decision making by taking into account a wider range of carbon emission sources. Not just the obvious and generic ones, leading to better overall carbon mitigation planning. As time progresses embodied carbon sources should be solidified with the aid of laws categorising and mandating the publishing of detailed embodied carbon emission data. Until this happens we need to proactively assess our carbon emitting actions during software development to help identify sources of carbon in software. We would hate for all our hard work in making software more sustainable to go to waste because there is an unknown source of carbon emissions silently harming the planet.
```
Comment (kietSL) [inline _posts/2023-09-26-embodied-carbon-from-software-development.md]: "As the time progresses..." I had to read this sentence multiple times to follow, maybe break it up with comma between "aid of laws" and "categorising".

Or "aid of laws which categorise and mandate the publishing..."

**Comment by kietSL** [comment]
That's an interesting read @jhowlett-scottlogic , well done! just a few niggles :)

File: _posts/2023-09-26-embodied-carbon-from-software-development.md (hunk 2 above)
Comment (jhowlett-scottlogic) [reply _posts/2023-09-26-embodied-carbon-from-software-development.md]: modified to help readability

File: _posts/2023-09-26-embodied-carbon-from-software-development.md (hunk 1 above)
Comment (jhowlett-scottlogic) [reply _posts/2023-09-26-embodied-carbon-from-software-development.md]: added

**Comment by kietSL** [comment]
Thanks for addressing comments, this looks good to me now. Just needs a date change before publishing @ColinEberhardt :)

**Pull request merged**